Cargo.toml

```diff
@@ -19,7 +19,7 @@ exclude = [
 
 [dependencies]
 itertools = { version = "0.12.1", default-features = false }
-unicode-width = "0.1"
+unicode-width = "0.1.12"
```

Comment:
Personally I prefer the 0.1 syntax over specifying the bugfix version here to as the lock file takes care of this and 0.1.11 will still install 0.1.12 regardless so it hides the fact that it allows for higher bugfix versions.